aodntools/timeseries_products/Documentation/Hourly_timeseries.md

```diff
@@ -1,22 +1,16 @@
 # Hourly Time Series Product (non-velocity)
 
-- [Objective](#objective)
+- [Output](#output)
 - [Scope](#scope)
 - [Output](#output)
 - [Method](#method)
 - [Distribution](#distribution)
 
 
 
-## Objective
-
-The project will provide 
-
-- A comprehensive set of aggregated files for each IMOS mooring site parameters (excluding current velocity data) binned into one hour intervals, excluding out-of-water data and records not flagged as “good” or “probably good” in the input files. QC flags will not be included. Statistics related to the averaging process will be stored as variables (standard deviation, minimum and maximum values, number of records averaged)
-- An additional version of the previous product that also includes the non quality controlled variables (qc flag 0 -- no_QC_performed) in the aggregated files.
-- All the (Python) code used for the generation of the products openly available on GitHub.
-- Clear documentation on the code and the output file format. 
+## Output
 
+The code will provide aggregated files for each IMOS mooring site parameters (excluding current velocity data) binned into one hour intervals, excluding out-of-water data and records not flagged as “good” or “probably good” in the input files. QC flags will not be included. Statistics related to the averaging process will be stored as variables (standard deviation, minimum and maximum values, number of records binned). An additional version of the product will be generated with the inclusion of the non quality controlled variables (qc flag 0 -- no_QC_performed).
```

Comment:
The last sentence should begin "A version of the product _can_ be generated...", otherwise it suggests the code creates two versions by default, which is not true.